suite-native/module-device-settings/src/screens/DeviceAutoConnectScreen.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, Card, PictogramTitleHeader, VStack } from '@suite-native/atoms';
+import { Translation, TxKeyPath } from '@suite-native/intl';
+import { DynamicScreenHeader, Screen } from '@suite-native/navigation';
+
+import { useDeviceAutoConnect } from '../hooks/useDeviceAutoConnect';
+
+type CardConfig = {
+    pictogramVariant: 'success' | 'info';
+    pictogramIcon: 'check' | 'x';
```

Comment:
nitpick, feel free to resolve: 
```suggestion
    pictogramVariant: PictogramVariant;
    pictogramIcon: IconName;
```

(I like when these configs are super easy to extend, but I don't imagine this one ever will extend, so you can resolve straight away should you want.)

Reply:
I was exact on purpose. Anyway, it seems this screen is going to be removed soon.